src/pages/fundamentals/eutxo-patterns.mdx

```diff
@@ -65,24 +273,102 @@ It is often useful to have a mutable state which either changes with each transa
 basis. One way to ensure that a datum is not 'spoofed' is to ensure that the input or reference input with
 that datum contains an NFT which has been generated to be unique using one of the method described above.
 
-## Actor Model ??
```

Comment:
Let's just remove this line if we are not going to address it at this time, I can loop back to it at a later date in the near future.